pkg/cmd/server/start/start_controllers.go

```diff
@@ -0,0 +1,76 @@
+package start
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/golang/glog"
+	"github.com/spf13/cobra"
+
+	kerrors "k8s.io/kubernetes/pkg/api/errors"
+	kcmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+)
+
+const controllersLong = `Start the master controllers
+
+This command starts the controllers for the master.  Running
+
+  $ %[1]s start master %[2]s
+
+will start the controllers that manage the master state, including the scheduler. The controllers
+will run in the foreground until you terminate the process.`
+
+// NewCommandStartMasterControllers starts only the controllers
+func NewCommandStartMasterControllers(name, fullName string, out io.Writer) (*cobra.Command, *MasterOptions) {
+	options := &MasterOptions{Output: out}
+	options.DefaultsFromName(fullName)
+
+	cmd := &cobra.Command{
+		Use:   "controllers",
+		Short: "Launch master controllers",
+		Long:  fmt.Sprintf(controllersLong, fullName, name),
+		Run: func(c *cobra.Command, args []string) {
+			if err := options.Complete(); err != nil {
+				fmt.Fprintln(c.Out(), kcmdutil.UsageError(c, err.Error()))
+				return
+			}
+
+			if len(options.ConfigFile) == 0 {
+				fmt.Fprintln(c.Out(), kcmdutil.UsageError(c, "--config is required for this command"))
+				return
+			}
+
+			if err := options.Validate(args); err != nil {
+				fmt.Fprintln(c.Out(), kcmdutil.UsageError(c, err.Error()))
+				return
+			}
+
+			startProfiler()
+
+			if err := options.StartMaster(); err != nil {
+				if kerrors.IsInvalid(err) {
+					if details := err.(*kerrors.StatusError).ErrStatus.Details; details != nil {
+						fmt.Fprintf(c.Out(), "Invalid %s %s\n", details.Kind, details.Name)
+						for _, cause := range details.Causes {
+							fmt.Fprintf(c.Out(), "  %s: %s\n", cause.Field, cause.Message)
+						}
+						os.Exit(255)
+					}
+				}
+				glog.Fatal(err)
+			}
+		},
+	}
+
+	options.MasterArgs = NewDefaultMasterArgs()
+	options.MasterArgs.StartAPI = false
+
+	flags := cmd.Flags()
+	flags.StringVar(&options.ConfigFile, "config", "", "Location of the master configuration file to run from. Required")
+	BindListenArg(options.MasterArgs.ListenArg, flags, "")
```

Comment:
It is, because we need an address to listen on for health check and metrics.  It's unused now but that's in the checklist.  It can't be the value from master because you might run controllers on the same host and you can't bind to the same name.